DNS records changes

DaMandal0rian opened the pull request:
- add continuim.cc zone
- add subdomains for subspace.network
- renaming resources to follow convention

vedhavyas commented on resources/dns/continuim-records.tf:
```diff
@@ -0,0 +1,17 @@
+resource "cloudflare_record" "main" {
```
maybe replace main with contunuim_cc ?

vedhavyas commented on resources/dns/records.tf:
```diff
@@ -75,7 +71,40 @@ resource "cloudflare_record" "uptime" {
+}
+
+// subspace Grafana monitor
+resource "cloudflare_record" "grafana" {
```
If we have grafana exposed, do we have the auth setup on grafana ?

DaMandal0rian replied:
https://grafana.subspace.network/

vedhavyas reviewed (commented):
Make sense. Do we have auth enabled for grafana and other services we are exposing here ?

DaMandal0rian committed:
DNS records changes

- add continuim.cc
- add subdomains for subspace.network
- renaming resources to follow convention
- minor change resource name
